src/_support/Page/Acceptance/Administrator/UserNotesFormPage.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * @package     Joomla.Test
+ * @subpackage  AcceptanceTester.Page
+ *
+ * @copyright   Copyright (C) 2005 - 2017 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+namespace Page\Acceptance\Administrator;
+
+/**
+ * Acceptance Page object class to user form page.
+ *
+ * @category  Users
+ * @package   Page\Acceptance\Administrator
+ * @author    Samarth sharma <samarthsharma351@gmail.com>
```

Comment:
Please remove author.  Only permitted in non-Joomla source files

Reply:
done